### DISCUSSION TITLE
Behavior is strange when multiple tabs are named in Japanese.

### DISCUSSION THREAD

**Discussion by lz-nakazawa** [Laravel-Backpack/community-forum#825] [Q&A (Help)] [answered]
https://backpackforlaravel.com/docs/6.x/crud-fields#optional-tab-attribute-splits-forms-into-tabs

If you prepare multiple tabs and give them Japanese names, each tab should have fields and you should be able to switch tabs, but the fields are grouped together on one tab.
I think this is probably because when using Japanese, anchor links are treated as blank characters.

I think it would be nice if there was an option to set the tab name and link separately. What do you think?

Is it possible for you to let me know if there is any smart solution at this point?

I'm Japanese and can't speak English well, so I apologize if it's hard to understand.

**Comment by pxpm** [comment, marked as answer]
Hey @lz-nakazawa your english is totally understandable no worries.

We had a similar problem a few time ago with hebrew that we fixed. https://github.com/Laravel-Backpack/CRUD/pull/5272

It's possible that you need to update your backpack version ? You can check your current version with `php artisan backpack:version`, I think the fix started on 6.1.7. (we are in 6.6.1 for reference). 

If you have the updated version and it still not working for you, please provide me the strings you are trying to use as slugs so that I can try to reproduce and come up with a solution.

Thanks, 🙏

**Reply by lz-nakazawa** [reply]
Hello! @pxpm 

I checked and found that I was using an older version of the backpack.
That is correct. Using the latest version (6.6.1) works fine!!

I'm embarrassed that I seem to have overlooked it.

Thank you for your prompt reply.
